src/server/serverBase.ts

```diff
@@ -83,7 +90,9 @@ export const handleReadFileRequest = (req: express.Request, res: express.Respons
     }
 };
 
+// tslint:disable-next-line:cyclomatic-complexity
 const findMatchingFile = (filePath: string, fileNames: string[], expectedFileName: string): string => {
+    let errorsCaught = 0;
```

Comment:
did you consider using https://github.com/isaacs/node-glob#readme to search file instead of going through all in for loop?

Reply:
Thanks for sharing this lib. It seems like it can find files with matching patterns, but I am not able to find if this library support check if a file in valid json format, do you have experience here?

Reply:
`glob("json/*.json", function(err, files)` will give you all json files In the path

Reply:
As discussed, I will mark this as resolved.